tests/integration/targets/dcnm_network/tests/dcnm/query.yaml

```diff
@@ -103,20 +103,60 @@
 - assert:
     that:
     - 'result.changed == false'
-    - 'result.response[0].gw_ip_subnet == "192.168.30.1/24"'
-    - 'result.response[0].net_extension_template == "Default_Network_Extension_Universal"'
-    - 'result.response[0].net_id == 7005'
-    - 'result.response[0].net_name == "ansible-net13"'
-    - 'result.response[0].net_template == "Default_Network_Universal"'
-    - 'result.response[0].vlan_id == "1500"'
-    - 'result.response[0].vrf_name == "Tenant-1"'
-    - 'result.response[1].gw_ip_subnet == "192.168.40.1/24"'
-    - 'result.response[1].net_extension_template == "Default_Network_Extension_Universal"'
-    - 'result.response[1].net_id == 7002'
-    - 'result.response[1].net_name == "ansible-net12"'
-    - 'result.response[1].net_template == "Default_Network_Universal"'
-    - 'result.response[1].vlan_id == "151"'
-    - 'result.response[1].vrf_name == "Tenant-2"'
+    - 'result.response[0].parent.networkName == "ansible-net13"'
```

Comment:
The structure that gets returned for query is quite different the the old structure and I no longer see some of the information in the test that we had previously.  What about `gw_ip_subnet`?  Do we still get that information in the query?

If we have users that are already using query but the structure of the dictionary has changed we might break existing playbooks.  Can we return the same structure and key names using the new query logic?

Reply:
We have not changed anything from the respone point of view, we are infact giving more response as part of the new logic.
The response contains the gw_ip_subent but this is inside a list of list of dict, i dont want to go inside the nested list for getting those values for the query IT cases, any how those fields have been tested and checked extensively in other state UT cases.